### ISSUE TITLE
Spelling/language errors and cleanliness is substandard

### ISSUE THREAD

**Issue by WilliamDEdwards** [crowdsecurity/crowdsec-docs#678]
As a possible new user, I notice that the documentation contains quite a few spelling/language errors throughout, and the general cleanliness is substandard.

For example, in the [HAProxy bouncer documentation](https://docs.crowdsec.net/u/bouncers/haproxy/) alone:

* "A lua Remediation Component for haproxy." -> incorrect capitalisation
* "This component leverages haproxy lua's API to check e IP address against the local API." -> incorrect capitalisation, random 'e'
* 'CrowdSec' is inconsistently capitalised as 'Crowdsec' and 'crowdsec' and 'CrowdSec'
* "Validated ips" -> incorrect capitalisation
* "you're good to go !" -> random space

These are just some examples, but examples like these can be found throughout the documentation. Although the documentation seems to be quite complete contents-wise, the sheer amount of English could scare off possible new users.

**Comment by LaurenceJJones** [comment]
> These are just some examples, but examples like these can be found throughout the documentation. Although the documentation seems to be quite complete contents-wise, the sheer amount of English could scare off possible new users.

Yeah we agree, the documentation is maintained by native and non native English speakers, we do use grammarly to try and spot these before commitment but obviously it no silver bullet.

> CrowdSec' is inconsistently capitalised as 'Crowdsec' and 'crowdsec' and 'CrowdSec'

Yes this is something we need to do a "wide" sed on, since the start of the year we decided that we should use proper capitalism so we need to retroactively go back through our older documentation.

Thank you for these spots, we will work on these!
